addons/default/visiosoft/profile-module/src/Seed/UsersFieldsSeeder.php

```diff
@@ -106,7 +91,7 @@ public function run(
                 'type' => 'anomaly.field_type.select',
                 'config' => [
                     'default_value' => 1,
-                    'options' => [0 => 'Active', 1 => 'Passive'],
+                    'options' => "0: Active\n1: Passive",
```

Comment:
is it right way? @diashalabi @ozcan-durak

Reply:
yes the old way was throwing an error. the options value should be a text not an array